packages/react-instantsearch-native/src/index.js

```diff
@@ -29,6 +29,7 @@ export { connectSortBy } from 'react-instantsearch-core';
 export { connectStateResults } from 'react-instantsearch-core';
 export { connectStats } from 'react-instantsearch-core';
 export { connectToggleRefinement } from 'react-instantsearch-core';
+export { connectHitInsights } from 'react-instantsearch-core';
```

Comment:
The imports are sorted, easier to navigate inside them.

Reply:
fixed in 99c3b1f